src/plot_data.jl

```diff
@@ -389,7 +401,7 @@ function combine_categories(
     keep_names = []
     for k in names
         if !isempty(data[k])
-            push!(values, agg(Matrix(data[k])))
+            push!(values, agg(Matrix(no_datetime(data[k]))))
```

Comment:
What's agg? It seems this is not a Julia method. If it is ours can we rename it, please?